aether-kernel/setup.cfg

```diff
@@ -14,4 +14,4 @@ omit = *migrations*, *tests*
 [coverage:report]
 omit = *migrations*, *tests*, *settings.py
 show_missing = true
-fail_under = 100
+fail_under = 95
```

Comment:
One step back...

Reply:
Ha! True. So what do you think? Should we accept 95%, or write the extra tests?

Reply:
We can accept 95% but I would suggest to write the missing tests in a near future.

Reply:
Agreed. Can you approve then please?